cli/cmd/inject.go

```diff
@@ -170,6 +170,11 @@ func (rt resourceTransformerInject) transform(bytes []byte) ([]byte, []inject.Re
 
 	reports := []inject.Report{*report}
 
+	if conf.IsService() {
+		b, err := conf.AnnotateService(rt.overrideAnnotations)
```

Comment:
I think we want to limit this to only set annotations which are applicable to services.  For now, this is only the opaque ports annotation.  Otherwise, calls like `linkerd inject --proxy-log-level=debug` can set annotations on the service which do not apply to service resources.